eng/Baseline.Designer.props

```diff
@@ -1004,9 +1004,9 @@
     <BaselinePackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[2.2.0, )" />
     <BaselinePackageReference Include="Microsoft.Extensions.Options" Version="[2.2.0, )" />
     <BaselinePackageReference Include="Microsoft.Net.Http.Headers" Version="[2.2.0, )" />
-    <BaselinePackageReference Include="System.Memory" Version="[4.5.1, )" />
+    <BaselinePackageReference Include="System.Memory" Version="[4.5.2, )" />
```

Comment:
Changes to this file shouldn't be merged.

Reply:
D'oh. I caught the other similar changes but I missed one it seems.